tests/build/build.go

```diff
@@ -505,16 +505,26 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 
 				for i := range gatherResult {
 					if gatherResult[i] == "sanity-inspect-image" {
+						// Fetching HACBS_TEST_OUTPUT shouldn't fail
 						result, err := build.FetchImageTaskRunResult(pipelineRun, gatherResult[i], "BASE_IMAGE")
 						Expect(err).ShouldNot(HaveOccurred())
 						ret := build.ValidateImageTaskRunResults(gatherResult[i], result)
 						Expect(ret).Should(BeTrue())
-						// TODO conftest-clair returns SUCCESS which is not expected
-						// } else {
-						// 	result, err := build.FetchTaskRunResult(pipelineRun, gatherResult[i], "HACBS_TEST_OUTPUT")
-						// 	Expect(err).ShouldNot(HaveOccurred())
-						// 	ret := build.ValidateTaskRunResults(gatherResult[i], result)
-						// 	Expect(ret).Should(BeTrue())
+					}
+					if gatherResult[i] == "conftest-clair" {
+						// Fetching HACBS_TEST_OUTPUT shouldn't fail
+						result, err := build.FetchTaskRunResult(pipelineRun, gatherResult[i], "HACBS_TEST_OUTPUT")
+						Expect(err).ShouldNot(HaveOccurred())
+						ret := build.ValidateTaskRunResults(gatherResult[i], result)
+						// Vulnerabilities should get periodically eliminated with image rebuild, so the result of that task might be different
+						// This should not block e2e tests with errors.
+						GinkgoWriter.Printf("retcode for validate taskrun result is %s\n", ret)
+					} else {
+						// Fetching HACBS_TEST_OUTPUT shouldn't fail
+						result, err := build.FetchTaskRunResult(pipelineRun, gatherResult[i], "HACBS_TEST_OUTPUT")
+						Expect(err).ShouldNot(HaveOccurred())
+						ret := build.ValidateTaskRunResults(gatherResult[i], result)
+						Expect(ret).Should(BeTrue())
```

Comment:
what about the result from [`sbom-json-check`](https://github.com/redhat-appstudio/e2e-tests/blob/21f6113671cd3438d02070a5fbe86097e86c1238/tests/build/build.go#L501)? should it be validated too?

Reply:
Yes. But that will be in a separate JIRA for adding that check.

Reply:
I think we can add the check as part of this change if it's not a big change.

Reply:
Done